exercises/lesson_03/README.rst

```diff
@@ -0,0 +1,69 @@
+==========
+Assignment
+==========
+
+In this assignment, you're going to practice locating files in your system as
+well as compression methods.
+
+1) Find all manual pages in your system which contain the string ``et`` in
+   their name (Note: manual pages are files located somewhere in your system).
+2) Copy the manual pages found in the previous step into the
+   ``mans_with_et_in_name`` directory in your home directory. Collect the names
+   of these manual pages, sort them alphabetically and write the resulting list
+   to the ``mans_with_et_in_name.txt`` file. The file contents should be
```

Comment:
You should specify where to place the .txt file. To home dir or inside created dir?